### PR TITLE
net-proxy/shadowsocks-libev: Bumps version to 2.1.4 and add live version

### PR DESCRIPTION
**Pull request by zhustec**

### CODE REVIEW

File: net-proxy/shadowsocks-libev/shadowsocks-libev-2.1.4.ebuild [hunk 1]
```diff
@@ -9,12 +9,18 @@ inherit eutils systemd
...
+	EGIT_REPO_URI="https://github.com/shadowsocks/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/shadowsocks/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
```
Comment (tolbkni) [inline net-proxy/shadowsocks-libev/shadowsocks-libev-2.1.4.ebuild:17]: Why remove this line?

Comment (zhustec) [inline net-proxy/shadowsocks-libev/shadowsocks-libev-2.1.4.ebuild:17]: Because I think live version don't need this.

Comment (biergaizi) [inline net-proxy/shadowsocks-libev/shadowsocks-libev-2.1.4.ebuild:17]: Yes.

Comment (biergaizi) [inline net-proxy/shadowsocks-libev/shadowsocks-libev-2.1.4.ebuild:17]: It acts as a mask.

**Comment by biergaizi** [comment]
Well done :+1:

**Pull request merged**